tests/validation/tests/v3_api/resource/terraform/k3s/master/join_k3s_master.sh

```diff
@@ -19,16 +19,31 @@ fi
 
 if [[ -n "${10}" ]] && [[ "${10}" == *"protect-kernel-defaults"* ]]
 then
-  cat /tmp/cis_masterconfig.yaml >> /etc/rancher/k3s/config.yaml
+  mkdir -p /var/lib/rancher/k3s/server/manifests
+  if [[ "${10}" == *"selinux"* ]]
+  then
+    yum -y install bc
```

Comment:
Same here: We should only install this based on OS, not necessarily if the selinux param is passed in config.